rskj-core/src/main/java/org/ethereum/rpc/CallArguments.java

```diff
@@ -0,0 +1,102 @@
+package org.ethereum.rpc;
+
+public class CallArguments {
+    private String from;
+    private String to;
+    private String gas;
+    private String gasLimit;
+    private String gasPrice;
+    private String value;
+    private String data; // compiledCode
+    private String nonce;
+    private String chainId; //NOSONAR
+
+    public String getFrom() {
+        return from;
+    }
+
+    public void setFrom(String from) {
+        this.from = from;
+    }
+
+    public String getTo() {
+        return to;
+    }
+
+    public void setTo(String to) {
+        this.to = to;
+    }
+
+    public String getGas() {
+        return gas;
+    }
+
+    public void setGas(String gas) {
+        this.gas = gas;
+    }
+
+    public String getGasLimit() {
+        return gasLimit;
+    }
+
+    public void setGasLimit(String gasLimit) {
+        this.gasLimit = gasLimit;
+    }
+
+    public String getGasPrice() {
+        return gasPrice;
+    }
+
+    public void setGasPrice(String gasPrice) {
+        this.gasPrice = gasPrice;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    public String getData() {
+        return data;
+    }
+
+    public void setData(String data) {
+        this.data = data;
+    }
+
+    public String getNonce() {
+        return nonce;
+    }
+
+    public void setNonce(String nonce) {
+        this.nonce = nonce;
+    }
+
+    public String getChainId() {
+        return chainId;
+    }
+
+    public void setChainId(String chainId) {
+        this.chainId = chainId;
+    }
+
+
+
```

Comment:
extra NLs ?)

Reply:
didn't get this one, NLs?

Reply:
I meant extra ***new lines***. Not that important though..

Reply:
ohhhh i see, not important but will fix it :D